apps/src/templates/sectionsRefresh/CurriculumQuickAssign.jsx

```diff
@@ -57,33 +59,102 @@ export default function CurriculumQuickAssign({
   }, [participantType]);
 
   useEffect(() => {
-    if (!courseOfferings) return;
+    // Filter the offerings based on the filters provided
+    const filterOfferings = data => {
```

Comment:
It is tough to filter by locale! The reason why is that the locale info is kinda buried inside the data. This does that, though... filters it by language (locale code).